api/src/main/java/org/openmrs/api/db/hibernate/search/LuceneAnalyzers.java

```diff
@@ -26,4 +26,6 @@ private LuceneAnalyzers() {
 	public static final String EXACT_ANALYZER = "exactAnalyzer";
 	
 	public static final String PHRASE_ANALYZER = "phraseAnalyzer";
+	
+	public static final String SOUNDEX_ANALYZER = "soundexExactAnalyzer";
```

Comment:
"soundexExactAnalyzer" sounds like a weird descriptor to me... SOUNDEX is, by its very nature, not an exact match, unless I'm missing something?

Reply:
Yes makes sense.  I will change it accordingly